packages/manager/src/features/IAM/Shared/Permissions/Permissions.style.ts

```diff
@@ -0,0 +1,71 @@
+import { Box, Typography } from '@linode/ui';
+import { Grid } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+export const sxTooltipIcon = {
+  marginLeft: 1,
+  padding: 0,
+};
+
+export const StyledTypography = styled(Typography, {
+  label: 'StyledTypography',
+})(({ theme }) => ({
+  color:
+    theme.name === 'light'
+      ? theme.tokens.color.Neutrals[90]
+      : theme.tokens.color.Neutrals.Black,
+  fontFamily: theme.font.bold,
+  marginBottom: 0,
+  paddingLeft: theme.spacing(0.5),
+}));
+
+export const StyledGrid = styled(Grid, { label: 'StyledGrid' })(() => ({
+  alignItems: 'center',
+  marginBottom: 0,
+}));
+
+export const StyledListItem = styled(Typography, { label: 'StyledListItem' })(
```

Comment:
Be sure to import this type from `@linode/ui` above as well as rename this like we did in the component file
```suggestion
export const StyledListItem = styled(Typography, { label: 'StyledListItem' })<TypographyProps>(
```